File: src/main/java/site/gachontable/gachontablebe/domain/admin/exception/SeatingNumAlreadyExistsException.java
package site.gachontable.gachontablebe.domain.admin.exception;

import site.gachontable.gachontablebe.global.error.ErrorCode;
import site.gachontable.gachontablebe.global.error.exception.ServiceException;

public class SeatingNumAlreadyExistsException extends ServiceException {
    public SeatingNumAlreadyExistsException() {
        super(ErrorCode.SEATING_NUM_ALREADY_EXIST);
    }
}

File: src/main/java/site/gachontable/domain/admin/exception/SeatingNumAlreadyExistsException.kt
package site.gachontable.domain.admin.exception;

import site.gachontable.independent.type.ErrorCode;
import site.gachontable.independent.error.exception.ServiceException;

public class SeatingNumAlreadyExistsException extends ServiceException {
    public SeatingNumAlreadyExistsException() {
        super(ErrorCode.SEATING_NUM_ALREADY_EXIST);
    }
}

